vxquery-xtest/results/xqts.txt

```diff
@@ -9511,7 +9511,7 @@ Expressions/PrimaryExpr/ParenExpr//Parenexpr-18, EXPECTED_RESULT_GOT_SAME_RESULT
 Expressions/PrimaryExpr/ParenExpr//Parenexpr-19, EXPECTED_RESULT_GOT_FAILURE
 Expressions/PrimaryExpr/ParenExpr//Parenexpr-2, EXPECTED_RESULT_GOT_SAME_RESULT
 Expressions/PrimaryExpr/ParenExpr//Parenexpr-20, EXPECTED_RESULT_GOT_ERROR
-Expressions/PrimaryExpr/ParenExpr//Parenexpr-3, EXPECTED_RESULT_GOT_SAME_RESULT
+Expressions/PrimaryExpr/ParenExpr//Parenexpr-3, EXPECTED_RESULT_GOT_DIFFERENT_RESULT
```

Comment:
I have no idea why this happened because i run the query and the result is correct...i updated again the xqts file and now the result is the same......I maybe posted an outdated file before.